src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/AssessmentEntity.kt

```diff
@@ -42,8 +44,16 @@ data class AssessmentEntity(
   val allocatedAt: OffsetDateTime,
 
   val createdAt: OffsetDateTime,
+
   var submittedAt: OffsetDateTime?,
+  @Enumerated(value = EnumType.STRING)
+  var decision: AssessmentDecision?,
 
   @Transient
   var schemaUpToDate: Boolean
 )
+
+enum class AssessmentDecision {
+  SUITABLE,
+  UNSUITABLE
```

Comment:
would `ACCEPTED | REJECTED` be more a natural pair of values for the outcome of assessing an application? Or are we thinking that this is a an assessment of a person's suitability for placement?

One of the reasons for a negative outcome would be that the applicant has failed to provide answers to follow up requests for further information. That makes me think that ACCEPTED/REJECTED is a better choice as what we ultimately need to record is a decision on the application rather than the suitability of the person being referred...

Reply:
makes sense 👍